ibis/sql/alchemy.py

```diff
@@ -564,6 +564,10 @@ def _window(t, expr):
         arg = _cumulative_to_window(t, arg, window)
         return t.translate(arg)
 
+    if window.max_lookback is not None:
+        raise NotImplementedError('Rows with max lookback is not implemented '
+                                  'for sqlalchemy backends.')
```

Comment:
Let's make this `SQLAlchemy-based backends`

Reply:
Fixed in next commit